doctr/datasets/synthtext.py

```diff
@@ -61,8 +69,8 @@ def __init__(
             if not os.path.exists(os.path.join(tmp_root, img_path[0])):
                 raise FileNotFoundError(f"unable to locate {os.path.join(tmp_root, img_path[0])}")
 
-            labels = [elt for word in txt.tolist() for elt in word.split()]
-            word_boxes = word_boxes.transpose(2, 1, 0) if word_boxes.ndim == 3 else np.expand_dims(word_boxes, axis=0)
+            labels = ''.join(txt).split()
+            word_boxes = word_boxes.transpose(2, 1, 0)
```

Comment:
@fg-mindee 
I think you cannot change this lines please try to load the dataset after downloading :)